interchain-token-service/src/state.rs

```diff
@@ -0,0 +1,133 @@
+use std::collections::HashMap;
+
+use axelar_wasm_std::IntoContractError;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, StdError, Storage};
+use cw_storage_plus::{Item, Map};
+use router_api::{Address, ChainName, ChainNameRaw};
+
+#[derive(thiserror::Error, Debug, IntoContractError)]
+pub enum Error {
+    #[error(transparent)]
+    Std(#[from] StdError),
+    #[error("ITS contract got into an invalid state, its config is missing")]
+    MissingConfig,
+    #[error("its address for chain {0} not found")]
+    ItsAddressNotFound(ChainName),
+}
+
+#[cw_serde]
+pub struct Config {
+    pub chain_name: ChainNameRaw,
```

Comment:
```suggestion
    pub chain_name: ChainName,
```

Reply:
Also, where is this used?

Reply:
Same question. What is this?

Reply:
The chain name of Axelar. It'll require special handling if ITS calls are received to Axelar itself. I'll remove it for now and add it back when needed.